pwiz_tools/Skyline/FileUI/FileUIResources.ja.resx

```diff
@@ -659,4 +659,7 @@ Old Localized:選択したフォルダには、Agilent QQQメソッドテンプ
   <data name="SkypSupport_ShowDownloadError_There_was_an_error_downloading_the_Skyline_document__0__from__1__" xml:space="preserve">
     <value>Skylineドキュメント{0}の{1}からのダウンロード中にエラーが発生しました。</value>
   </data>
+  <data name="OpenFileDialogNE_OpenFileDialogNE_Open" xml:space="preserve">
+	  <value>開く(&amp;O)</value>
```

Comment:
Sorry I didn't notice before. These should be spaces (you may have to change the Visual Studio settings to make it use spaces in these xml files). You should be able to "Amend previous commit" to avoid having to squash again. Same issue with the other two resx files.

Reply:
Sorry didn't see way to amend last commit in TortoiseGIT so added commit and squashed again.
I will look at how to amend later.

I fixed the spacing in the FileUI... resx files.

I didn't see any spacing issues in the Base... resx files.
I only changed "openButton" to "actionButton" so should not have changed any spacing.